webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -18,6 +18,8 @@ export class SakaiRubricGrading extends rubricsApiMixin(RubricsElement) {
     this.criteria = [];
     this.totalPoints = 0;
 
+    this.uniqueKey = Math.floor(Math.random() * 100);
```

Comment:
While unlikely to repeat in this case, it doesn't guarantee uniqueness. I've seen `Math.floor(Date.now() * Math.random())` used for extra comfort.